### PR TITLE
Updated singly linked list

### PR DESCRIPTION
**Pull request by taher27**
1. default constructor was missing. 
   so it is now created at line 143.
2. error: use data instead of value.
 that change is also updated now.

### CODE REVIEW

**Comment by taher27** [comment]
Node InsertNth(Node head, int data, int position) {
        
        Node newNode = new Node();
        newNode.data = data;             // old code
        newNode.value = data;		// updated code
        
        if (position == 0) {
            newNode.next = head;

public Node(int valuein){
		value = valuein;
	}

**Comment by taher27** [comment]
and Add default constructor

Default constructor
	public Node(){
		value = 0;    
	}

File: DataStructures/Lists/SinglyLinkedList.java [hunk 1]
```diff
@@ -46,7 +48,7 @@ public void insertHead(int x){
     Node InsertNth(Node head, int data, int position) {
         
         Node newNode = new Node();
-        newNode.data = data;
+        newNode.value = data;		// take value
```
Comment (taher27) [inline DataStructures/Lists/SinglyLinkedList.java]: Updated code from variable name ( data to value)

File: DataStructures/Lists/SinglyLinkedList.java [hunk 2]
```diff
@@ -141,6 +143,13 @@ public Node(int valuein){
 		value = valuein;
 	}
 
+
+
+// Default constructor
+	public Node(){
```
Comment (taher27) [inline DataStructures/Lists/SinglyLinkedList.java]: add 1 default constructor.

**Comment by shellhub** [comment]
Closed because of Conflicting

**Pull request closed without merging**